apps/admin-gui/src/app/shared/components/member-overview-membership/member-overview-membership.component.html

```diff
@@ -18,15 +18,14 @@ <h1 class="page-subtitle">
           <th *matHeaderCellDef mat-header-cell></th>
           <td *matCellDef="let attribute" class="column-center" mat-cell>
             <div *ngIf="attribute === 'Status'">
-              <i
-                class="material-icons vert-center mr-1 {{(member.status | memberStatusIconColor)}}">
+              <i class="material-icons vert-center mr-1 {{(member | memberStatusIconColor)}}">
                 {{member.status | memberStatusIcon}}
               </i>
               <b class="vert-top">
                 {{member.status | transformMemberStatus}}
               </b>
               <button
-                *ngIf="authResolver.isThisVoAdmin(vo.id)"
+                *ngIf="authResolver.isThisVoAdmin(vo.id) && !(member | memberListCheckboxDisabled)"
```

Comment:
Change the selector of this pipe (or maybe even refactor the name) so it better explains what it is actually used for. Here it definitely does not disable checkbox.